task/buildah-oci-ta/0.4/buildah-oci-ta.yaml

```diff
@@ -110,6 +110,13 @@ spec:
         with remote VMs
       type: string
       default: "false"
```

Comment:
Can we have also option to enable/disable reusing images? It will be useful for tests and to unblock users in case of bugs in the feature (given the size of feature, no way that there won't be bugs)

Reply:
this has been added as a task param.